docs/pages/guides/customizing-metro.md

```diff
@@ -1,11 +1,18 @@
 ---
-title: Customizing Metro
+title: Metro bundler
+sidebar_title: Bundling with Metro
 ---
 
 import { Terminal } from '~/ui/components/Snippet';
 import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
 
-When you run `npx expo start`, the [Expo CLI][expo-cli] uses [Metro](https://facebook.github.io/metro/) to bundle JavaScript for Android and iOS platforms. By default [Expo CLI][expo-cli] will use the Metro configuration defined in the [`@expo/metro-config`](https://github.com/expo/expo/tree/main/packages/@expo/metro-config) package. You can add custom options for Metro by creating a file named **metro.config.js** in the project root directory or by running:
+Expo CLI uses Metro during [`npx expo start`](/workflow/expo-cli#develop) and [`npx expo export`](/workflow/expo-cli#exporting) to bundle your JavaScript code and assets. Metro is built and optimized for React Native, and used for large scale applications like Facebook and Instagram.
+
+## Customizing
+
+You can customize the Metro bundler by creating a **metro.config.js** file in the root of your project. This file should export a [Metro configuration][metro-config] that extends [`@expo/metro-config`][expo-metro-config].
```

Comment:
we should explain here why we import `expo/metro-config` rather than `@expo/metro-config`

Reply:
```suggestion
You can customize the Metro bundler by creating a **metro.config.js** file in the root of your project. This file should export a [Metro configuration][metro-config] that extends [`expo/metro-config`][expo-metro-config]. Users should import `expo/metro-config` instead of `@expo/metro-config` to ensure version consistency.
```